disco_aws_automation/disco_elastigroup.py

```diff
@@ -218,6 +249,12 @@ def _create_az_subnets_dict(self, subnets):
             zones[subnet['AvailabilityZone']] = subnet['SubnetId']
         return zones
 
+    def wait_for_instance_id(self, group_name):
+        """Wait for instance id(s) of an elastigroup to become available"""
+        while not all([instance['instance_id'] for instance in self.get_instances(group_name=group_name)]):
+            logger.info('Waiting for instance id(s) of %s to become available', group_name)
+            time.sleep(10)
```

Comment:
This is not a backoff due to restrictions on frequency of API calls; this is merely a periodic check on availability of instance id.